dashboard/api.py

```diff
@@ -592,5 +598,20 @@ def refresh_user_data(user_id):
         try:
             CachedEdxDataApi.update_cache_if_expired(user, edx_client, cache_type)
         except:
+            save_cache_update_failure(user_id)
```

Comment:
I'm not sure we want to call this function in both places, if there is an exception raised it would have a different reason for each case. We should handle only the specific error we expect, which is in the above except clause. Can you remove this line?

Reply:
this is  for this error https://sentry.io/mit-office-of-digital-learning/micromasters/issues/228392933/